package.json

```diff
@@ -151,7 +151,7 @@
     "build": "cross-env NODE_ENV=production webpack --display-error-details --verbose --progress",
     "run": "cross-env NODE_ENV=production webpack --progress --watch",
     "run-fullmap": "webpack --progress --watch",
-    "test": "jest",
+    "test": "jest --forceExit",
```

Comment:
`jest` was refusing to exit on Jenkins, googling led me to other people who had the problem suggesting this